data/test/macros/start_position_common_keep_a_b_c_d.cfg

```diff
@@ -13,6 +13,13 @@
 # There is no free castle hex to recruit onto.
 ##
 #define COMMON_KEEP_A_B_C_D_UNIT_TEST NAME CONTENT
+
+#arg SIDE1_TEAM
+"A"#endarg
+
+#arg SIDE2_TEAM
+"B"#endarg
```

Comment:
These arguments never seem to be used (by the places using this macro). I'm not opposed to having them, just that they don't seem to be used in this PR.

Having the default names be longer would be easier for searching for places that they're used, but this is where having code using the functionality would help judge whether the defaults are good.

Reply:
> just noticed that `SIDE2_TEAM` is used

An alternative would be to use this in two tests, which I'd prefer over having more macro arguments.
```
        [modify_side]
            side=1,2
            team_name=allies
        [/modify_side]
```

Reply:
Updated.

Reply:
A copy of the same is still in start_position_generic.cfg though.

Reply:
Updated.